distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Date_Time.enso

```diff
@@ -572,9 +584,10 @@ type Date_Time
        Produces a warning if the resulting date time is before an Enso epoch.
 
        Arguments:
-       - amount: The amount of time to subtract from this instant. Can be either
-         `Duration` for subtracting time (hours, minutes, etc.), or `Period`
-         for subtracting days, months and years.
+       - amount: The amount of time to subtract from this instant, either
+       `Duration` for adding time (hours, minutes, etc.), `Period` for adding
+       date (days, months, years), or `Date_Period` for adding a single unit
+       (year, quarter, month, etc.).
```

Comment:
We also allow adding a `Time_Period`. So it feels like this variant should be reflected in the comment as well.

Reply:
Done.